source/frontend/src/tenants/config/defaultTenant.ts

```diff
@@ -109,6 +115,70 @@ export const defaultTenant: ITenantConfig2 = {
     CDOGS:
       'The DevExchange Document Generation Service is experiencing service degradation, you will be unable to generate form documents (ie. H120, H1005) until resolved.',
   },
+  LandTitleDistricts: [
```

Comment:
nit: please rename field name to follow JSON conventions -> "landTitleDistricts"